README.md

```diff
@@ -1,9 +1,29 @@
-# WordPress-Android-Editor
+# WordPress-Editor-Android #
 
-## Introduction
+[![Build Status](https://travis-ci.org/wordpress-mobile/WordPress-Editor-Android.svg?branch=develop)](https://travis-ci.org/wordpress-mobile/WordPress-Editor-Android)
 
-The WordPress-Android-Editor is the text editor used in the [WordPress Android app](https://github.com/wordpress-mobile/WordPress-Android) to create and edit pages & posts. In short it's a simple, straightforward way to visually edit HTML.
+## Introduction ##
 
-## LICENSE
+WordPress-Editor-Android is the text editor used in the [WordPress Android app](https://github.com/wordpress-mobile/WordPress-Android) to create and edit pages & posts. In short it's a simple, straightforward way to visually edit HTML.
 
-This library is licensed under[MIT](LICENSE-MIT)
+## Build Instructions ##
+
+Post-checkout instructions for Windows, necessary to convert the assets symlink to a Windows symlink:
+
+From git bash, inside the cloned project root:
+
+    $ rm WordPressEditor/src/main/assets
+    $ git ls-files --deleted -z | git update-index --assume-unchanged -z --stdin
+
+Then, from a Windows command prompt:
+
+    mklink /D [PROJECT_ROOT]\WordPressEditor\src\main\assets %PROJECT_ROOT%\libs\editor-common\assets
+
+Finally, update `[PROJECT_ROOT]\.git\info\exclude` to ignore the symlink locally:
+
+    # assets symlink
+    WordPressEditor/src/main/assets
+
+## LICENSE ##
+
+This library is licensed under [MIT](LICENSE-MIT).
```

Comment:
Let's keep this consistent with wpandoid...[GPLv2](https://github.com/wordpress-mobile/WordPress-Android#license)